src/devices/Arduino/samples/DebugLogStream.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
```

Comment:
can we simplify the sample and print directly to console rather than having this class?

Reply:
Removed this class for now. It is only useful for very specific low-level testing.